src/main/java/com/adobe/campaign/tests/logparser/core/LogData.java

```diff
@@ -490,6 +491,74 @@ public File exportLogDataToHTML(Collection<String> in_headerSet, String in_repor
         return l_exportFile;
     }
 
+    /**
+     * Exports the current LogData to a standard JSON file. By default, the file will have an escape version of the Parse
+     * Definition as the name
+     *
+     * @return a JSON file containing the LogData
+     */
+    public File exportLogDataToJSON() throws LogDataExportToFileException {
+        T l_firstEntry = this.fetchFirst();
+
+        if (l_firstEntry != null) {
+            return exportLogDataToJSON(l_firstEntry.fetchHeaders(), l_firstEntry.getParseDefinition().fetchEscapedTitle() + "-export.json");
+        } else {
+            log.warn("No Log data to export. Please load the log data before re-attempting");
+            return null;
+        }
+    }
+
+    /**
+     * Exports the current LogData to a standard JSON file.
+     *
+     * @param in_jsonFileName a filename to store the CSV export
+     * @return a JSON file containing the LogData
+     */
+    public File exportLogDataToJSON(String in_jsonFileName) throws LogDataExportToFileException {
+        T l_firstEntry = this.fetchFirst();
+
+        if (l_firstEntry != null) {
+            return exportLogDataToJSON(l_firstEntry.fetchHeaders(), in_jsonFileName + "-export.json");
+        } else {
+            log.warn("No Log data to export. Please load the log data before re-attempting");
+            return null;
+        }
+    }
+
+    /**
+     * Exports the current LogData to an JSON file
+     *
+     * @param in_headerSet   A set of headers to be used as keys for exporting
+     * @param in_jsonFileName The file name to export
+     * @return a JSON file containing the LogData
+     * @throws LogDataExportToFileException If the file could not be exported
+     */
+    public File exportLogDataToJSON(Collection<String> in_headerSet, String in_jsonFileName) throws LogDataExportToFileException {
+        File l_exportFile = LogParserFileUtils.createNewFile(in_jsonFileName);
+        List<Map<String, String>> JSONlist = new ArrayList<>();
```

Comment:
A local variable should not start with capital letters.

Reply:
Variable renamed